Examples/S3EventNotifier/README.md

```diff
@@ -0,0 +1,52 @@
+# S3 Event Notifier
+
+This example demonstrates how to create a Lambda that notifies an API of an S3 event in a bucket.
+
+## Code
+
+In this example the lambda function receives an `S3Event` object from the `AWSLambdaEvents` library as input object instead of a `APIGatewayV2Request`. The `S3Event` object contains all the information about the S3 event that triggered the lambda, but what we are interested in is the bucket name and the object key, which are inside of a notification `Record`. The object contains an array of records, however since the lambda is triggered by a single event, we can safely assume that there is only one record in the array: the first one. Inside of this record, we can find the bucket name and the object key:
```

Comment:
The S3Event is defined in the Lambda event library but the event is sent by S3.
I would not refer to APIGatewayRequest - this is not bringing new value in the context of this example.

"the lambda" -> "the Lambda function" or just "the function" (2x)